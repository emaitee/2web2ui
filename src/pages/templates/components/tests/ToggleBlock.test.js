import { shallow } from 'enzyme';
import React from 'react';

import ToggleBlock from '../ToggleBlock';

describe('ToggleBlock', () => {
  let wrapper;

  beforeEach(() => {
    const props = {
      input: {
        name: 'name'
      },
      meta: {},
      label: 'label',
      helpText: 'some help'
    };

    wrapper = shallow(<ToggleBlock {...props} />);
  });

  afterEach(() => {
    wrapper.unmount();
  });

  it('should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
