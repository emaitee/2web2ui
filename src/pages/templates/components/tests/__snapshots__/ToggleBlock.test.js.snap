// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleBlock should render 1`] = `
<div>
  <Grid>
    <Grid.Column
      xs={8}
    >
      <label>
        label
      </label>
    </Grid.Column>
    <Grid.Column
      xs={4}
    >
      <div>
        <Toggle
          id="name"
          name="name"
        />
      </div>
    </Grid.Column>
  </Grid>
  <p>
    some help
  </p>
</div>
`;
